src/polytope/models/SourceCode.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+from enum import Enum, auto
+
+
+class SourceCodeLanguage(Enum):
+    """! Languages of source codes enumeration class."""
+
+    # Unix shell bash.
+    Bash = auto()
+    # C11 (gcc)
+    C11 = auto()
+    # C++20 (g++)
+    Cpp20 = auto()
+    # Python3
+    Python3 = auto()
```

Comment:
Do we need to mention Specific version of python? Python3_7 / Python3_7_14...

Reply:
That sounds like a lot better!
For now, it seems sufficient to mention only the minor version.

https://github.com/Studio-Polytope/Polytope/blob/940bf14f32bf3c02c87d656bdfbbcbec2db826ba/src/polytope/models/SourceCode.py#L5-L18